src/app/static/src/app/components/figures/filteredData.ts

```diff
@@ -24,16 +20,27 @@ export function useFiltering(props: FilteringProps) {
                     return acc
                 }, {} as Dictionary<string | number>)
         }
+        return settings;
+    })
+
+    const filterBySettings = (row: any) => {
+
+        const settingsVal = settings.value;
+        if (!settingsVal) {
+            return true;
+        }
 
-        for (let key of Object.keys(settings)) {
-            if (row[key] != undefined && (settings[key] === "" || row[key] != settings[key])) {
+        for (let key of Object.keys(settingsVal)) {
+            if (row[key] != undefined &&
+                (settingsVal[key] === "" || (row[key] != settingsVal[key] && row[key] != "n/a"))) {
```

Comment:
Here's the actual logic change - return rows that match settings OR that have the value "n/a".